### ISSUE TITLE
Block aggregations of RCN and Comcast Fiber to tract level to map

### ISSUE THREAD

**Issue by theryankelly** [MAPC/digital-equity-report#38]

**Comment by theryankelly** [comment]
@mapc-lberman Thank you for putting these maps together. K:\DataServices\Projects\Current_Projects\EconDev\Digital_Divide\Data\Spatial\Block_Level_Fiber_Data_2022

The plan is to have one map of the available fiber in the report. This map will live in the [Executive Summary](https://mapc.github.io/digital-equity-report/report/#summary) and in [Existing Conditions: Infrastructure](https://mapc.github.io/digital-equity-report/report/#existing-conditions). While the by-provider information is helpful for us to see, I think we want to highlight the full picture of fiber from these residential providers together. This will mean pulling into now regional map.

I have been weighing what buffer is most appropriate. I think we can go with a 200ft / 61m buffer. However the current bucketing is a bit confusing. Adding percentages to the legend rather than decimals might help, but what are thoughts on fewer categories? Something feels off when I ask "_which blocks have less than half of their area near fiber?_" Although, seeing the percentage in a low grouping is helpful, "_those that have less than 10% of the block is near a fiber_". Would this benefit from being an interactive map with a slider filter? Let users see what areas have a low % of fiber coverage? Thoughts @mapc-lberman @japartridge @trust-kimberly @TimReardon 

- [ ] Please combine Comcast and RCN maps into one regional map with 200 ft buffer
- [ ] Legend labels to percentages 

![comcast_100ft_vs_200ft](https://user-images.githubusercontent.com/2421382/159075369-5b484e7d-66ae-4c26-ad58-014338f164f0.gif)
![RCN_100ft_vs_200ft](https://user-images.githubusercontent.com/2421382/159075379-e7520c2e-256f-4cdc-9c6f-dc31bb73ed46.gif)

**Comment by trust-kimberly** [comment]
I agree it would be more helpful to see the legend in percents. I'm not sure how these buckets were originally weighted but it feels more digestible to break down the range into quintiles. We lose a bit of specificity in having fewer, even groupings but I'm not sure that the current legend is getting that specificity across. 

Whether we make this interactive or static, it might also be helpful to include percent of total blocks that fall within each range. As a very rough estimate for the sake of this example, somewhere we should display "40% of our communities have less than 10% of the block near fiber", "20% of our communities have 10-20% of the block near fiber", etc.

**Comment by mapc-lberman** [comment]
hi, currently I'm working on the ACS processing tasks, but I can switch back to the Broadband maps if that is needed.   Can you confirm that you want me to combine both the RCN and Comcast into a single access map?  and you want quintiles as percents?  (note I would just multiply the pct col by 100 in the latter case.) cc-ing jesse on slack... thanks.

**Comment by theryankelly** [comment]
@mapc-lberman can confirm that will want you to create a combined map. Would like to hear from others about bucketing before proceeding.

**Comment by mapc-lberman** [comment]
roger that

**Comment by japartridge** [comment]
i agree with all points made. for the buckets, we could try quintiles and round to whole/even numbers. Lex and i can look at it together this week.

**Comment by theryankelly** [comment]
Closing the loop on this viz. We are going to go with a map that is 200 ft and buckets of 0, 1 to 38, 39 to 64, 65 to 90, 91 to 99, 100. Changed the title to **Proximity to Comcast and RCN Fiber** and added direct labeling of "%" to the legend. Also added a description to help explain.

**Description:**
@mapc-lberman I think we can drop the ~% from the description and just say:
_The percentage ranges indicate the area of each block that lies within 200 ft of either Comcast or RCN fiber lines_ 


**Maybe in the report add description text like:**
Currently, only 21% of all census blocks in the region are fully within 200ft of a Comcast or RCN fiber line.
Or 
Fiber is not ubiquitous in the region. Currently, 79% of census blocks in these cities have sections, not with 200 ft of a Comcast or RCN fiber line. 319 census blocks or 18.7% have not fiber running through them from these providers.

0 class = 319 out of  1700  =  18.7%
1 to 38 = 279 out of 1700 = 16.5 %
39 to 64  = 278 out of 1700 = 16.3 %
65 to 90 = 286 out of 1700 =  16.8 %
91 to 99 = 179 out of 1700 = 10.5 %
100 = 359 out of 1700 = 21.1 %

![Proximity_200ft_Fiber_map_2022-04-07](https://user-images.githubusercontent.com/2421382/162495085-1a44e46b-6bf8-4cc9-a87c-df1d1ba2b0dd.jpg)

**Comment by mapc-lberman** [comment]
update of map with final percentage bins [0, 1 to 38, 39 to 64, 65 to 90, 91 to 99, 100] and text changes.

![Proximity_200ft_Fiber_map_2022-04-08](https://user-images.githubusercontent.com/91992267/162515341-0a789c67-0a86-4872-893c-dbb6baf540c5.jpg)
